python/ray/data/tests/test_binary.py

```diff
@@ -195,6 +198,64 @@ def skip_unpartitioned(kv_dict):
     )
 
 
+def _gen_chunked_binary(
+    dir_path: str, total_size: int, max_file_size: Optional[int] = None
```

Comment:
Do we ever configure the `max_file_size`?

Reply:
Was also using this util for other test that's gonna be added in this PR (for Canva issue)